cobalt/web/url.cc

```diff
@@ -69,6 +110,21 @@ std::string URL::CreateObjectURL(
     return "";
   }
 
+  web::EnvironmentSettings* web_settings =
+      base::polymorphic_downcast<web::EnvironmentSettings*>(
+          environment_settings);
+  DCHECK(web_settings);
+  DCHECK(web_settings->context());
+  web::WindowOrWorkerGlobalScope* global_scope =
+      web_settings->context()->GetWindowOrWorkerGlobalScope();
+  if (global_scope->IsWorker() &&
+      !(IsMseInWorkersEnabled(web_settings) &&
+        IsMediaElementUsingMediaSourceAttachmentMethodsEnabled(web_settings) &&
+        IsMediaElementUsingMediaSourceBufferedRangeEnabled(web_settings))) {
+    // Prevent usage of MSE-in-Workers if pre-requisites are not satisfied.
+    return "";
```

Comment:
Not very familiar with this, if URL is "", is this a behavior by default?

Only three H5VCC flags are true, then MSE-in-Workers can be enabled. If yes, should we check if `global_scope->IsWorker()` it true, then we check if any of the flags is false, then return ""?